src/routes/index.svelte

```diff
@@ -42,10 +68,63 @@
 </script>
 
 <style lang="scss">
+  @import "scss/site/mixins/media-queries";
+
+  .alert :global(.bx--inline-notification) {
+    margin: 0;
+    max-width: 100%;
+    background: $teal-10;
+
+    &::before {
+      border: none;
+    }
+  }
+
+  .alert :global(.bx--inline-notification__text-wrapper) {
+    flex-direction: column;
+  }
+
+  .bx--inline-notification__subtitle {
+    margin-bottom: 0;
+  }
+
   .btn-container {
     margin-top: 2.5rem;
     text-transform: uppercase;
   }
+
+  // The following code creates a grid that has 1/3 and 2/3 column template
+  .one-two-column-grid {
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(18rem, 1fr));
+    grid-gap: 2rem;
+    padding: 0;
+    margin: 0;
+  }
+
+  .ae-logo-box {
+    padding: 14px;
+
+    img {
+      display: block;
+      margin: auto;
+    }
+  }
+
+  .ae-logo-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    align-items: flex-start;
```

Comment:
Make this text fill the remaining space on larger screens and appear more compact on smaller screens add:

For larger screens:
add `grid-column: span 2;` 

For smaller screens:
```
.ae-logo-text {
  @include media("<=medium") {
     grid-column: span 1;
  }
}
```